Website/src/items/renderings/edgewebsite/News/NewsList.yml

```diff
@@ -4,38 +4,38 @@ Parent: "deb2e94a-9a21-4305-ad48-76704da0e6f1"
 Template: "04646a89-996f-4ee7-878a-ffdbf1f0ef0d"
 Path: /sitecore/layout/Renderings/Project/edgewebsite/News/NewsList
 SharedFields:
-- ID: "037fe404-dd19-4bf7-8e30-4dadf68b27b0"
-  Hint: componentName
-  Value: NewsList
-- ID: "06d5295c-ed2f-4a54-9bf2-26228d113318"
-  Hint: __Icon
-  Value: "edgeicons/32x32/news-list.png"
-- ID: "b0b15510-b138-470e-8f33-8da2e228aafe"
-  Hint: Rendering Contents Resolver
-  Value: "{4D45DA3E-8128-422C-AFDA-E10672117E15}"
+  - ID: "037fe404-dd19-4bf7-8e30-4dadf68b27b0"
```

Comment:
Please revert formatting changes made to this file. Serialized YAML items have a specific format that must not be changed. Please disable "format on save" option of your text editor if that was the cause of the change.